### PR TITLE
Day11

### PR DESCRIPTION
**Pull request by frunzisss**

### CODE REVIEW

File: js/Events/logInEvents.js [hunk 1]
```diff
@@ -19,9 +19,8 @@ const addEventsLogIn = function (){
         } else
         
         if(checkCredentials(userNameValue, passwordValue)){
-            const app = document.getElementById('app');
-            app.innerHTML = EvaluationsPage();
-            addEventsEvaluations();
+            RedirectTo('evaluations');
+            sessionStorage.setItem('loggedUser', userNameValue);
```
Comment (s-timofte) [inline js/Events/logInEvents.js:23]: really liked that you used sessionStorage for this

File: js/applicationManager.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+function RedirectTo(page){
+    const app = document.querySelector('#app');
+    
+    switch (page) {
```
Comment (s-timofte) [inline js/applicationManager.js:4]: a next step here would be to remove the events that are not common as well when moving to a new page

Comment (frunzisss) [reply js/applicationManager.js:4]: Sure thing :)

Comment (frunzisss) [reply js/applicationManager.js:4]: dar eu am o functie removeAll() pe care o apelez cand schimb pagina. Poate e confusing pentru ca o apelez din EventsCommon?

Comment (s-timofte) [reply js/applicationManager.js:4]: e confusing pt ca o apelezi din CommonEvents. Daca era din EventsCommon era less confusing. Daca era din EventUtils era perfect din pct meu de vedere :)

**Pull request merged**